src/main/kotlin/com/cognifide/gradle/aem/deploy/PurgeTask.kt

```diff
@@ -0,0 +1,29 @@
+package com.cognifide.gradle.aem.deploy
+
+import com.cognifide.gradle.aem.AemPlugin
+import org.gradle.api.tasks.TaskAction
+
+open class PurgeTask : AbstractTask() {
+
+    companion object {
+        val NAME = "aemPurge"
+    }
+
+    init {
+        group = AemPlugin.TASK_GROUP
+        description = "Uninstalls and then deletes AEM package on instance(s)."
+    }
+
+    @TaskAction
+    fun purge() {
+        deploy({ sync ->
+            propertyParser.checkForce()
+
+            val packagePath = determineRemotePackagePath(sync);
+
+            uninstallPackage(packagePath, sync)
```

Comment:
what if there will be no package here? we agreed that purge should continue even uninstallPackage will end with no package on install